### PR TITLE
Add data.frame generics to find_motifs & write_fasta

### PR DESCRIPTION
**Pull request by ErdaradunGaztea**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: tidysq
 Type: Package
 Title: Tidy Processing and Analysis of Biological Sequences
-Version: 1.1.3
+Version: 1.1.4
```
Comment (DominikRafacz) [inline DESCRIPTION:4]: I think this change should bump a minor, not only patch

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# tidysq 1.1.4
```
Comment (DominikRafacz) [inline NEWS.md:1]: version bump applies here too

File: R/write_fasta.R [hunk 3]
```diff
@@ -25,19 +28,51 @@
 #'              alphabet = "dna_bsc")
 #' write_fasta(sq_dna,
 #'             c("bat", "cat", "rat", "elephant_swallowed_by_A_snake"),
-#'             "~/fasta_rubbish/example.fasta")
```
Comment (DominikRafacz) [inline R/write_fasta.R:28]: oh wow, was it here for the entire time? ;D

File: src/RcppExports.cpp [hunk 4]
```diff
@@ -390,7 +390,7 @@ BEGIN_RCPP
 END_RCPP
 }
 
-RcppExport SEXP run_testthat_tests(SEXP use_xml_sxp);
+RcppExport SEXP run_testthat_tests();
```
Comment (DominikRafacz) [inline src/RcppExports.cpp:393]: Have you tried running the tests or doesn't that break CI tests? The call of some `testthat` internal function has changed but still was rendered incorrectly by Rcpp, that's why there is a difference in this parameter -- I've inserted that manually after generating `RcppExports.cpp`

Comment (ErdaradunGaztea) [reply src/RcppExports.cpp:393]: Everything seem to work now, so this workaround is no more necessary.

File: DESCRIPTION (hunk 1 above)
Comment (DominikRafacz) [reply DESCRIPTION]: done in new commit

File: NEWS.md (hunk 2 above)
Comment (DominikRafacz) [reply NEWS.md]: done in the new commit

**Comment by DominikRafacz** [comment]
@ErdaradunGaztea , I have bumped the version and also added alternative way of calling modified functions in the README and vignette

**Pull request merged**